### ISSUE TITLE
Import pypi packages 

### ISSUE THREAD

**Issue by sks444** [coala/community#114]
Create an app name `products` and import the list https://pypi.python.org/simple/ of available PyPI packages, and its related details like releases, author, download links, see https://pypi.python.org/simple/abstrackr/, https://pypi.python.org/pypi/abstrackr/json

priority/very low

**Comment by Makman2** [comment]
What do you need all package information for?

**Comment by Makman2** [comment]
Especially this is quite much info, requires some disk space/RAM^^

**Comment by sks444** [comment]
Yup; this is silly. Actually, I was thinking about the [org schema](https://github.com/coala/community/issues/113), so I thought it might be good to have some details about PyPi packages and its releases, but yeah that's a lot of data. So, the plan is to only get the packages of a particular org, the org who loads this app. i.e: in our case, we will get all the details of `coala`  PyPi products. But not sure how we are gonna get this for a specific org.

**Comment by Makman2** [comment]
Is there maybe other API we can use from PyPI to query for this information? Something like "search-by-package-name" or so?

About doing that for a specific org: You can probably look where our publishing accounts appear, maybe "Package Index Owner" or so.
